test/val/val_decoration_test.cpp

```diff
@@ -2219,6 +2219,38 @@ TEST_F(ValidateDecorations,
           "rules: member 1 at offset 4 is not aligned to 16"));
 }
 
+TEST_F(ValidateDecorations, PushConstantMissingBlockVulkanBad) {
```

Comment:
This test should succeed with a universal environment. Please modify this test to use the Vulkan environment and add a new test for the universal environment.